app/jobs/post_grouper.rb

```diff
@@ -0,0 +1,19 @@
+#class PostGrouper < ActiveJob::Base
+class PostGrouper
+  def perform(cleanup_gene_claims = true)
```

Comment:
default here should be false. We shouldn't be deleting gene or drug claims unless it's a special circumstance (e.g. _ensembl_).